local_dev/master_local_dev.js

```diff
@@ -39,10 +39,27 @@ var debugContext = {
     bindings: {
         req
     },
-    log: function () {
-        var util = require('util');
-        var val = util.format.apply(null, arguments);
-        console.log(val);
+    log: {
+        error : function() {
+            var util = require('util');
+            var val = util.format.apply(null, arguments);
+            return console.log('ERROR:', val);
+        },
+        warning : function() {
+            var util = require('util');
+            var val = util.format.apply(null, arguments);
+            return console.log('WARNING:', val);
+        },
+        info : function() {
+            var util = require('util');
+            var val = util.format.apply(null, arguments);
+            return console.log('INFO:', val);
+        },
+        verbose : function() {
+            var util = require('util');
+            var val = util.format.apply(null, arguments);
+            return console.log('VERBOSE:', val);
```

Comment:
Not very DRY.  Calling a common utility function with an argument might be slicker.

Reply:
agree with that - let's move all that common code to one place if possible

Reply:
Not sure how to achieve this. Any suggestions are welcome.
I'm continue looking for a better implementaion